src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,108 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exception.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationServiceImpl registrationService;
+    private static StorageDaoImpl storage;
+    private static final String VALID_LOGIN_1 = "validLogin1";
+    private static final String VALID_LOGIN_2 = "validLogin2";
+    private static final String EMPTY_LOGIN = "";
+    private static final String VALID_PASSWORD = "validPassword";
+    private static final String EMPTY_PASSWORD = "";
+    private static final String INVALID_LENGTH = "short";
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        storage = new StorageDaoImpl();
+
+    }
+
+    @Test
+    void validRegistration_Ok() {
```

Comment:
rename your methods so they adhere to the naming convention for this task
for example
register_validData_ok()